tests/integration/security/jwt_test.go

```diff
@@ -465,3 +468,182 @@ func TestRequestAuthentication(t *testing.T) {
 			}
 		})
 }
+
+// TestRequestAuthentication tests beta authn policy for jwt on ingress.
+// The policy is also set at global namespace, with authorization on ingressgateway.
+func TestRequestAuthenticationOnIngress(t *testing.T) {
+	framework.NewTest(t).
+		RequiresEnvironment(environment.Kube).
+		Run(func(ctx framework.TestContext) {
+			var ingr ingress.Instance
+			var err error
+			if ingr, err = ingress.New(ctx, ingress.Config{
+				Istio: ist,
+			}); err != nil {
+				t.Fatal(err)
+			}
+			t.Log(ingr)
+
+			ns := namespace.NewOrFail(t, ctx, namespace.Config{
+				Prefix: "req-authn",
```

Comment:
`req-authn-ingress` to avoid duplication with the `TestRequestAuthentication`

Reply:
done